app/lib/common/pages/medications/cubit.dart

```diff
@@ -1,3 +1,5 @@
+import 'dart:convert';
+
 import 'package:freezed_annotation/freezed_annotation.dart';
```

Comment:
```suggestion
import 'package:freezed_annotation/freezed_annotation.dart';
```

Remove this import and make a helper method in `models/medication.dart` similar to `medicationWithGuidelinesFromHTTPResponse`